spec/license_scout/dependency_manager/habitat_spec.rb

```diff
@@ -171,12 +171,12 @@
         expect(glibc.version).to eq("2.27-20180608041157")
         expect(glibc.license.records.first.id).to eql("GPL-2.0")
         expect(glibc.license.records.first.source).to eql("https://bldr.habitat.sh/v1/depot/channels/core/unstable/pkgs/glibc/2.27/20180608041157")
-        expect(subject.fetched_urls["core/glibc"]).to eql("https://bldr.habitat.sh/v1/depot/channels/core/stable/pkgs/glibc/2.27/20180608041157")
+        expect(subject.fetched_urls["core/glibc"]).to eql("https://bldr.habitat.sh/v1/depot/channels/core/froghornetsnest/pkgs/glibc/2.27/20180608041157")
```

Comment:
The comment on this test case (lines 159-161) makes me think that the original expectation was the /intent/ of the test here (that we would be pulling dependencies from the fallback origin) so I'm wondering if this assertion was correct but the test setup is somehow wrong?